<ui version="4.0" >
 <class>udg::QInputOutputLocalDatabaseWidgetBase</class>
 <widget class="QWidget" name="udg::QInputOutputLocalDatabaseWidgetBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <property name="horizontalSpacing" >
    <number>0</number>
   </property>
   <property name="verticalSpacing" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <widget class="QSplitter" name="m_StudyTreeSeriesListQSplitter" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="childrenCollapsible" >
      <bool>false</bool>
     </property>
     <widget class="udg::QStudyTreeWidget" native="1" name="m_studyTreeWidget" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="udg::QSeriesListWidget" native="1" name="m_seriesListWidget" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <property name="topMargin" >
      <number>6</number>
     </property>
     <item>
      <widget class="QToolButton" name="m_viewButton" >
       <property name="text" >
        <string>View</string>
       </property>
       <property name="icon" >
        <iconset resource="../main/main.qrc" >:/images/view.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolButtonStyle" >
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>udg::QStudyTreeWidget</class>
   <extends>QWidget</extends>
   <header>qstudytreewidget.h</header>
  </customwidget>
  <customwidget>
   <class>udg::QSeriesListWidget</class>
   <extends>QWidget</extends>
   <header>qserieslistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
